prov/gni/src/gnix_msg.c

```diff
@@ -518,6 +699,8 @@ static int __gnix_rndzv_req(void *arg)
 		_gnix_nic_tx_free(nic, txd);
 		GNIX_INFO(FI_LOG_EP_DATA, "GNI_PostRdma failed: %s\n",
 			  gni_err_str[status]);
+
```

Comment:
you don't want a GNIX_TRACE macro here.  If you want GNIX_DEBUG might be good though.